CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.23.6] - 11/28/2023
+
+- [818](https://github.com/OSC/ood_core/pull/818) Changes `cluster.batch_connect_ssh_allow?` default from `nil` to `true`
```

Comment:
Thanks for updating the changlog.  If I let github auto generate this, here are some others we could add.

I think all the real users could go here too, and it looks like we should definitely go to `0.24.0` because of utkarshayachit's changes.

## What's Changed
* Code cleanup and separate arguments with whitespace in Fujitsu TCS adapter by @mnakao in https://github.com/OSC/ood_core/pull/808
* Add OUT_OF_MEMORY state for Slurm by robinkar in https://github.com/OSC/ood_core/pull/809
* find_port: avoid infinite loop by utkarshayachit in https://github.com/OSC/ood_core/pull/811
* handle find_port error codes by utkarshayachit in https://github.com/OSC/ood_core/pull/812
* vnc: run websockify as background process by utkarshayachit in https://github.com/OSC/ood_core/pull/813
* Add working_dir option for Fujitsu TCS job scheduler by mnakao in https://github.com/OSC/ood_core/pull/816
* Minor fix for Fujitsu TCS by mnakao in https://github.com/OSC/ood_core/pull/817
* Update rake requirement from ~> 13.0.1 to ~> 13.1.0 by dependabot in https://github.com/OSC/ood_core/pull/814
* Changes default return value for cluster.batch_connect_ssh_allow? by HazelGrant in https://github.com/OSC/ood_core/pull/818

## New Contributors
* robinkar made their first contribution in https://github.com/OSC/ood_core/pull/809
* utkarshayachit made their first contribution in https://github.com/OSC/ood_core/pull/811
* HazelGrant made their first contribution in https://github.com/OSC/ood_core/pull/818

**Full Changelog**: https://github.com/OSC/ood_core/compare/v0.23.5...v0.24.0